pkg/logs/input/docker/launcher.go

```diff
@@ -210,6 +242,7 @@ func (l *Launcher) restartTailer(containerID string) {
 	oldTailer, exists := l.tailers[containerID]
 	if exists {
 		source = oldTailer.source
+		l.collectAllSource.RemoveInput(containerID)
```

Comment:
Do you still need to do this ? I am wondering  since you're already doing it in the tailer and you injected the status to the source

Reply:
Yes, because the inputs are link to a source and not to the status